src/app/css/_connectivityPane.scss

```diff
@@ -48,4 +48,14 @@
             padding-top: 50px;
         }
     }
-}
\ No newline at end of file
+}
+
+.connection-string-dropDown {
+    width: 95% !important;
+}
+
+.hidden {
```

Comment:
I think this is something that causes some HUGE accessibility nightmares of screens shifting focus.

Reply:
this is what we already do for masked text field: maskedCopyableTextField.tsx. I don't believe we got accesibility on portal. let me confirm with @c-ryan-k

Reply:
Yeah, we experienced a similar issue with Firefox, instead of shifting things a bunch of pixels away to hide them, you can leave them in place but make them 'effectively invisible':
`height: 1px;`
`width: 1px;`
`opacity: 0;`

and for screen readers, etc - the elements that are "hidden" should be marked with `aria-hidden="true"`

Reply:
updated to follow the best practice